internal/fields/runtime_field.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Nit. Avoid using underscores in Go files.

Reply:
I've updated this file and I took the advantage of renaming `dynamic_template.go` and `exception_fields.go` files too

Reply:
Changes done in https://github.com/elastic/elastic-package/pull/2415/commits/37f613c84cede484c8298d992066043372ab7b64